sound/soc/sof/intel/hda.c

```diff
@@ -452,8 +457,13 @@ void hda_dsp_dump(struct snd_sof_dev *sdev, u32 flags)
 		snd_sof_get_status(sdev, status, panic, &xoops, &panic_info,
 				   stack, HDA_DSP_STACK_DUMP_SIZE);
 	} else {
-		dev_err(sdev->dev, "error: status = 0x%8.8x panic = 0x%8.8x\n",
-			status, panic);
+		if (hda->boot_iteration == HDA_FW_BOOT_ATTEMPTS)
+			dev_err(sdev->dev, "error: status = 0x%8.8x panic = 0x%8.8x\n",
+				status, panic);
+		else
+			dev_dbg(sdev->dev, "status = 0x%8.8x panic = 0x%8.8x\n",
+				status, panic);
+
```

Comment:
... same here. But, but, we don't print the panic code anywhere else, so only printing a dbg here, or only printing the err on last iteration, both seem like bad options.

Reply:
Agree, I couldn't find a better option really.
This makes the ICL RVP CI errors go away though, which is a good thing.